masi-STP-backend/src/main/java/pl/lodz/p/it/masi/stp/chatbot/services/ReportServiceImpl.java

```diff
@@ -94,7 +94,7 @@ private void addQeiContent(Document document, List<ConversationLog> conversation
             for (int i = 0; i < conversationLog.getMessagesLogs().size(); i++) {
                 Paragraph messageQei = new Paragraph();
                 BigInteger resultCount = conversationLog.getMessagesLogs().get(i).getResultsCount();
-                if (!Objects.equals(resultCount, BigInteger.ZERO)) {
+                if (!Objects.equals(resultCount, BigInteger.ZERO) && resultCount != null) {
```

Comment:
Podoba mi się sposób w jaki rozwiązałeś zaistniały problem. Akceptuję.

Reply:
Dziękuję Szanowny Panie Inżynierze.